includes/class-migration.php

```diff
@@ -182,7 +182,7 @@ public static function maybe_migrate() {
 		if ( \version_compare( $version_from_db, '5.2.0', '<' ) ) {
 			Scheduler::register_schedules();
 		}
-		if ( \version_compare( $version_from_db, '5.3.0', '<' ) ) {
+		if ( \version_compare( $version_from_db, 'unreleased', '<' ) ) {
 			add_action( 'init', 'flush_rewrite_rules', 20 );
```

Comment:
@obenland why can't we call the `flush_rewrite_rules` directly?

Reply:
Because `Activitypub::add_rewrite_rules()` [runs after](https://github.com/Automattic/wordpress-activitypub/blob/5.3.2/includes/class-activitypub.php#L42) `maybe_migrate()`. Running it on 20 makes sure the rewrite rules have been registered.

Reply:
why I ask: https://github.com/Automattic/wordpress-activitypub/issues/1393